### PR TITLE
move import path according to repro name

### PR DESCRIPTION
**Pull request by ryanzhang-oss**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,15 +1,25 @@
-/.cache
-/.work
-/_output
-/config/
-/config
-cover.out
-/vendor
-/.vendor-new
```
Comment (negz) [inline .gitignore:8]: I think some or all of the above need to stay in the `.gitignore` as they're generated by the Makefile but shouldn't be checked in.

File: Makefile [hunk 2]
```diff
@@ -92,3 +92,23 @@ oam-runtime.help:
 help-special: oam-runtime.help
 
 .PHONY: oam-runtime.help help-special
+
+generate: controller-gen
+	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
```
Comment (negz) [inline Makefile:97]: I don't think we need this - the our build submodule make lib calls `go generate`, which calls this per https://github.com/crossplane/oam-kubernetes-runtime/blob/2cccfe365030ec41e27d78ef356f0fc9281816f3/apis/generate.go#L25

Comment (ryanzhang-oss) [reply Makefile:97]: how does this work? I think that the goal is to make this repro independent enough that anyone who is interested in OAM can come to contribute. I am trying to set up a generic system to achieve that and this is just the first step.

**Comment by negz** [comment]
@ryanzhang-oss Can you update the repo name in the `Jenkinsfile` too, please?

**Pull request merged**